wcfsetup/install/files/lib/system/user/notification/event/ModerationQueueCommentUserNotificationEvent.class.php

```diff
@@ -36,15 +36,16 @@ class ModerationQueueCommentUserNotificationEvent extends AbstractUserNotificati
     protected $languageItemPrefix = '';
 
     /**
-     * moderation queue object the notifications (indirectly) belong to
-     * @var ViewableModerationQueue
+     * language item for the type name
+     * @var string
      */
-    protected $moderationQueue;
+    protected $typeName = '';
```

Comment:
Likewise this should be typed without a default, especially since the default probably isn't useful it's better to error out if it is used uninitialized.